app.js

```diff
@@ -4,5 +4,10 @@ document.addEventListener("DOMContentLoaded", () => {
   const game = new Game();
 
   // Start game
-  game.start({ healthElement: ".health" });
+  game.start({
+    healthElement: document.querySelector(".health"),
```

Comment:
**suggestion:** Be careful with that change, because you changed argument type here from `string` to `HTMLElement`. Ensure, that you have adjusted the rest of your logic.